charts/kubeai/values.yaml

```diff
@@ -29,6 +29,11 @@ modelServers:
     images:
       default: "fedirz/faster-whisper-server:latest-cpu"
       nvidia-gpu: "fedirz/faster-whisper-server:latest-cuda"
+  Infinity:
+    images:
+      default: "michaelf34/infinity:latest"
+      cpu: "michaelf34/infinity:latest"
```

Comment:
You should need "default"

Reply:
fixed, good catch! It works fine with just default.